pubtools/_quay/container_image_pusher.py

```diff
@@ -110,15 +110,19 @@ def run_tag_images(cls, source_ref, dest_refs, all_arch, target_settings):
             target_settings.get("tag_images_wait_time_increase", 10),
         )
 
-    def copy_source_push_item(self, push_item):
+    def copy_source_or_v1_push_item(self, push_item, is_source=None):
         """
         Perform the tagging operation for a push item containing a source image.
 
         Args:
             push_item (ContainerPushItem):
                 Source container push item.
         """
-        LOG.info("Copying push item '{0}' as a source image".format(push_item))
+        if is_source:
+            LOG.info("Copying push item '{0}' as a source image".format(push_item))
+        else:
+            LOG.info("Copying push item '{0}' as v1 container only".format(push_item))
```

Comment:
So, what would happen during signing? Would V2S2 manifest be available? We want to sign V2S2 of these images, right?

Reply:
good point, we should make sure there's v2sch2 digest. I guess it should be similar to situation in https://github.com/release-engineering/pubtools-quay/pull/102